commons-persistence/src/main/resources/schema.sql

```diff
@@ -1585,239 +762,2048 @@ BEGIN
+
+ALTER TABLE ONLY public."VEX"
+    ADD CONSTRAINT "VEX_PROJECT_FK" FOREIGN KEY ("PROJECT_ID") REFERENCES public."PROJECT"("ID") DEFERRABLE INITIALLY DEFERRED;
+
+ALTER TABLE ONLY public."VIOLATIONANALYSISCOMMENT"
+    ADD CONSTRAINT "VIOLATIONANALYSISCOMMENT_VIOLATIONANALYSIS_FK" FOREIGN KEY ("VIOLATIONANALYSIS_ID") REFERENCES public."VIOLATIONANALYSIS"("ID") DEFERRABLE INITIALLY DEFERRED;
+
+ALTER TABLE ONLY public."VIOLATIONANALYSIS"
+    ADD CONSTRAINT "VIOLATIONANALYSIS_COMPONENT_FK" FOREIGN KEY ("COMPONENT_ID") REFERENCES public."COMPONENT"("ID") DEFERRABLE INITIALLY DEFERRED;
+
+ALTER TABLE ONLY public."VIOLATIONANALYSIS"
+    ADD CONSTRAINT "VIOLATIONANALYSIS_POLICYVIOLATION_FK" FOREIGN KEY ("POLICYVIOLATION_ID") REFERENCES public."POLICYVIOLATION"("ID") DEFERRABLE INITIALLY DEFERRED;
+
+ALTER TABLE ONLY public."VIOLATIONANALYSIS"
+    ADD CONSTRAINT "VIOLATIONANALYSIS_PROJECT_FK" FOREIGN KEY ("PROJECT_ID") REFERENCES public."PROJECT"("ID") DEFERRABLE INITIALLY DEFERRED;
+
+ALTER TABLE ONLY public."VULNERABILITIES_TAGS"
+    ADD CONSTRAINT "VULNERABILITIES_TAGS_TAG_FK" FOREIGN KEY ("TAG_ID") REFERENCES public."TAG"("ID") DEFERRABLE INITIALLY DEFERRED;
 
-INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('procedure_update-portfolio-metrics', 'nscuro@protonmail.com', 'migration/changelog-procedures.xml', NOW(), 35, '9:1de7336fd9c6d13ceaccf6a498492eed', 'createProcedure path=procedures/procedure_update-portfolio-metrics.sql', '', 'EXECUTED', NULL, NULL, '4.28.0', '9395800427');
+ALTER TABLE ONLY public."VULNERABILITIES_TAGS"
+    ADD CONSTRAINT "VULNERABILITIES_TAGS_VULNERABILITY_FK" FOREIGN KEY ("VULNERABILITY_ID") REFERENCES public."VULNERABILITY"("ID") DEFERRABLE INITIALLY DEFERRED;
 
--- Release Database Lock
-SET SEARCH_PATH TO public, "$user","public";
+ALTER TABLE ONLY public."VULNERABLESOFTWARE_VULNERABILITIES"
+    ADD CONSTRAINT "VULNERABLESOFTWARE_VULNERABILITIES_VULNERABILITY_FK" FOREIGN KEY ("VULNERABILITY_ID") REFERENCES public."VULNERABILITY"("ID") DEFERRABLE INITIALLY DEFERRED;
 
-UPDATE databasechangeloglock SET LOCKED = FALSE, LOCKEDBY = NULL, LOCKGRANTED = NULL WHERE ID = 1;
+ALTER TABLE ONLY public."VULNERABLESOFTWARE_VULNERABILITIES"
+    ADD CONSTRAINT "VULNERABLESOFTWARE_VULNERABILITIES_VULNERABLESOFTWARE_FK" FOREIGN KEY ("VULNERABLESOFTWARE_ID") REFERENCES public."VULNERABLESOFTWARE"("ID") DEFERRABLE INITIALLY DEFERRED;
 
-SET SEARCH_PATH TO public, "$user","public";
+ALTER TABLE ONLY public."WORKFLOW_STATE"
+    ADD CONSTRAINT "WORKFLOW_STATE_WORKFLOW_STATE_FK" FOREIGN KEY ("PARENT_STEP_ID") REFERENCES public."WORKFLOW_STATE"("ID") DEFERRABLE INITIALLY DEFERRED;
 
-INSERT INTO "CONFIGPROPERTY" ("GROUPNAME", "PROPERTYNAME", "PROPERTYVALUE", "PROPERTYTYPE", "DESCRIPTION")
-VALUES ('internal', 'cluster.id', '53b94f78-6e2f-4015-961a-0965ff1807a9', 'STRING', NULL);
```

Comment:
This part is not included in the schema dump, but is needed for the tests. Since usually the API server populates this upon startup.

Reply:
I was trying to figure out the same, thanks Niklas 👍🏼